nglib/CMakeLists.txt

```diff
@@ -43,4 +43,10 @@ if(USE_PYTHON)
 endif(USE_PYTHON)
 
 install(TARGETS nglib ${NG_INSTALL_DIR})
-install(FILES nglib.h DESTINATION ${NG_INSTALL_DIR_INCLUDE}/include COMPONENT netgen_devel)
+install(FILES nglib.h DESTINATION ${NG_INSTALL_DIR_INCLUDE}/nglib COMPONENT netgen_devel)
+install(FILES nglib.h DESTINATION ${NG_INSTALL_DIR_INCLUDE} COMPONENT netgen_devel)
+
```

Comment:
Why do you install nglib.h four times?

Reply:
I did this after some frustration with the cmake of FreeCAD. The proper solution is to change the FindNetgen used in FreeCAD.
But the place where it got placed originally is also quite strange... /include/netgen/include .

Reply:
Concerning the include dir structure: That's because we maintain the relative paths of the header files in the source code. The headers in 
libsrc/include contain statements like
```
#include "../gprim/gprim.hpp"
```
Therefore we need an additional "include" in the path.

Concerning the FindNetgen.cmake in FreeCAD: Are there reasons not to use NetgenConfig.cmake and netgen-targets.cmake provided by Netgen itself?

Reply:
I will have a look at the default cmake, but I think we need our custom FindNetgen, because we are also using older versions of netgen which was build without cmake...